src/metabase/models/setting.clj

```diff
@@ -406,6 +411,14 @@
   set to true when settings are being written directly via /api/setting endpoints."
   false)
 
+(defn- has-feature?
+  [feature]
+  (u/ignore-exceptions
+   (classloader/require 'metabase.public-settings.premium-features))
+  (if-let [has-feature?' (resolve 'metabase.public-settings.premium-features/has-feature?)]
```

Comment:
I don't think this can return falsy, maybe just call it directly without an `if`